android/src/main/java/com/cronjob/BootReceiver.java

```diff
@@ -82,4 +82,4 @@ public void onReceive(Context context, Intent i) {
 
     }
 
-}
```

Comment:
I did not specifically edited this bit, should be just a EOL change from the github online IDE